### DISCUSSION TITLE
FM band scan. with automatic storage of FM stations

### DISCUSSION THREAD

**Discussion by infra-81** [PE5PVB/TEF6686_ESP32#616] [Ideas]
1. create an option (command).
to scan the FM band and automatically store fm radio stations
 (allows you to quickly store FM radio stations when traveling)

2. another option (command). to erase all radio memories

**Comment by Yurij20** [comment]
I’ve written about this idea before and I hasten to remind you about it, because I think it’s important for traveling in order to more conveniently study the frequency plan of the city you’re visiting.  To do this, you need to enter: 1. Auto-search for radio stations of the current range with auto-saving them into memory.  2. Auto-sorting of memory channels in order of increasing frequency with their compaction and filling of deleted memory cells as a result of removing phantom frequencies after auto-scanning.  3. Completely delete all saved stations from memory.  If you start the auto search without clearing all the memory, the auto search fills the remaining free memory cells.  This functionality is perfectly implemented in one of the modifications of firmware 1.15.  I can attach a link if necessary.

**Comment by PE5PVB** [comment]
Done.
